### DISCUSSION TITLE
Github Education Verification Failures

### DISCUSSION THREAD

**Discussion by Cupara** [community/community#51791] [GitHub Education]
I have attempted to verify 4 times, and of the 4, 3 times were rejected. The 4th went through finally but I fear it will be rejected as the unofficial transcript I used in the photo doesn't have my graduation date on it. I have a screenshot of my active class schedule that has my expected graduation date in it. I'm an online learner so the location verification process shouldn't take that into account as most universities do not provide any proof of being a distant learner as well distant learner means different terms in the US as it's used for special needs children attending public schools. Those of us learning online prefer to be called Online Learners. Anyway, I want to get this screenshot shared with my application before it gets rejected. My classes require me to use Github all the time for every weekly assignment and final project so I want to have full access to all the resources Github has to offer but that is going to be hard getting rejected once again.

Thanks

**Comment by Ntshangase** [comment]
Hi,

You don't need to have the graduation date on the documents, but they need to show that you are registered for this year. A proof of registration with this year's date should be good.

**Reply by Cupara** [reply]
Thank you, it does show I'm registered for this year.

**Reply by Ntshangase** [reply]
Maybe using a **VPN** can help since you stay far from your campus.

This discussion sound similar to yours: https://github.com/orgs/community/discussions/46668

**Reply by Cupara** [reply]
I tried a VPN but it's like the system could detect I wasn't near a campus still. I'm not sure what else to do at this point.

**Comment by samus-aran** [comment]
Hi @Mythotical 

Let's see how we can assist ⭐ 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

For remote student applicants: please ensure that your evidence indicates that you are a remote student. If possible, we encourage students to apply whilst on campus. This will assist us in confirming your student status.

Hope this helps! Let us know how you get along 👍

**Reply by Cupara** [reply]
Nope, that doesn't help. I live too far away to visit a campus otherwise I would just go to the campus and do all of this. I have an academic email but that is the reason evidence is required. I have tried with various types of evidence and still get denied each and every time. I'm not sure what else I can do.

Any other suggestions?

**Comment by Cupara** [comment]
I now have a verification letter from the school on their letterhead that states I'm in the online program which means I'm a distant learner but their system won't accept it. Still states I'm not near a campus but yet my letter states I'm a distant learner. This whole automated system needs to be revamped or bring back the old system where manual approval was needed in situations such as mine. I'm not sure what else to do from here. I've attached my verification letter for support here just in case they can manually approve me.

[Enrollment Verification - SMoore1.pdf](https://github.com/community/community/files/11333474/Enrollment.Verification.-.SMoore1.pdf)

**Reply by samus-aran** [reply]
Hi @Mythotical 

Can you provide the latest rejection reasons you've received 👍

**Reply by Cupara** [reply]
Sure, here is a screenshot of it.

![Request-a-discount-GitHub-Education](https://user-images.githubusercontent.com/7572268/234729111-96c456be-a757-4e8c-b719-55133f89ca2e.png)

**Reply by Cupara** [reply]
> Hi @Mythotical
> 
> Can you provide the latest rejection reasons you've received 👍

Any update to this?

**Comment by snasif** [comment]
I'm having the same issues, I don't know how to find help. My reason for being rejected are:

You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.

You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

I'm an online student and I've updated my profile mulitple times. Not sure what else to do.

**Reply by Cupara** [reply]
> I'm having the same issues, I don't know how to find help. My reason for being rejected are:
> 
> You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.
> 
> You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.
> 
> I'm an online student and I've updated my profile mulitple times. Not sure what else to do.

You need to make sure you full name on your GitHub profile matches how your school lists it otherwise you will get denied for that reason.

With the campus thing, I'm still trying to get this taken care of considering I provided a letter from my college on their letterhead stating I'm an online student.

**Reply by snasif** [reply]
The name matches exactly, but I keep getting denied. I emailed support and they said to come here and find help lol.

**Reply by snasif** [reply]
@Mythotical 

After 13 failed attempts, I'm finaly approved. Using my phone I took a photo of my laptop of the school portal showing my schedule and my name.

**Reply by Cupara** [reply]
> @Mythotical
> 
> After 13 failed attempts, I'm finaly approved. Using my phone I took a photo of my laptop of the school portal showing my schedule and my name.

I'll give it a try later and see what happens.

**Comment by facesnorth** [comment]
What an abomination this verification system is.  Nearly up there with the most frustrating things I've ever experienced in tech and automated customer service in my 45 years of life.

**Reply by josemartinez111** [reply]
Exactly what i said. They should just eliminate it if they don't want to provide a decent way to get enrolled. I've literally wasted hours trying to upload my id, transcripts, screen shots of any page that matters, changed my user name, added a brief bio etc... Nothing. It now also states I'm not near the school but obviously I'm an online student. So now I have to verify that too? 😂🥱

**Reply by facesnorth** [reply]
SOMEHOW.....after dozens and dozens of attempts across weeks, doing the same thing over and over repeatedly, they JUST NOW let met in....finally.

What I did was I went to the "My study" tab on my student portal using chrome (it may be different for you depending on your school), I clicked ctrl-shift-i and then ctrl-shift-p and typed "screenshot", then I clicked "capture full sized screenshot".  it downloaded a full sized PNG.  then I converted the PNG to JPG using a website service.  I uploaded this file and selected the 2nd option from the top, I believe it was "dated transcript" or along those lines.  (At first I had tried the bottom option, screenshot of student portal, and this was rejected).  I also needed to use NordVPN to connect to London since my school is based there, and github kept saying I can't possibly be a student there (idiots).  Then I uploaded it and it worked.  Finally.  So extremely annoying.

**Comment by Cupara** [comment]
I just used my phone to take a photo of my portal and it worked. Woohoo.
